### ISSUE TITLE
Incorrect computation of line width with multibyte characters

### ISSUE THREAD

**Issue by echasnovski** [JohnnyMorganz/StyLua#577]
1. Create a file with line `string.rep("ыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыы", 10)`.
2. Run `stylua` with default config on that file. It results into multi line function call, but line width is only 70 characters, **but** 122 bytes.

Version: 0.14.3.

**Comment by JohnnyMorganz** [comment]
Wherever we do `.len()` on a string we should probably compute graphemes. Will probably need to bring in an external crate for this

**Comment by echasnovski** [comment]
Yeah... I feel you. Multibyte characters are a constant source of trouble.
